Projects/App/Sources/UI/Organization/OrgDetail/OrgDetailViewController.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  OrgDetailView.swift
+//  App
+//
+//  Created by Keum MinSeok on 2022/10/12.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+
+import UIKit
+import DesignSystem
+import SnapKit
+
+final class OrgDetailViewController: UIViewController {
+
+    // MARK: - Properties
+
+    private let orgNameLabel = UILabel()
+    private let orgInformationSuperStackView = UIStackView()
+    private let orgInformationStackView1 = UIStackView()
+    private let orgInformationStackView2 = UIStackView()
+    private let orgInformationStackView3 = UIStackView()
+    private let orgInformationImage1 = UIImageView()
+    private let orgInformationImage2 = UIImageView()
+    private let orgInformationImage3 = UIImageView()
+    private let orgInviteButton = FullWidthBlackButton()
+
+    private var orgDetailInformationView1 = OrgDetailInformationView()
+    private var orgDetailInformationView2 = OrgDetailInformationView()
+    private var orgDetailInformationView3 = OrgDetailInformationView()
+
+    // MARK: - LifeCycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+        createLayout()
+    }
+
+    // MARK: - Function
+
+}
+
+// MARK: - UI Function
+
+extension OrgDetailViewController {
+    
+    private func configureUI() {
+        view.backgroundColor = .zestyColor(.background)
+        
+        orgDetailInformationView1 = OrgDetailInformationView()
+        orgDetailInformationView2 = OrgDetailInformationView()
+        orgDetailInformationView3 = OrgDetailInformationView()
+        
+        orgNameLabel.text = "애플 디벨로퍼 아카데미"
+        orgNameLabel.textColor = .black
+        orgNameLabel.font = UIFont.systemFont(ofSize: CGFloat(26), weight: .bold)
+        orgNameLabel.textAlignment = .center
+        orgNameLabel.numberOfLines = 2
+        
+        orgInformationSuperStackView.axis = .vertical
+        orgInformationSuperStackView.alignment = .top
+        orgInformationSuperStackView.distribution = . fillProportionally
+        orgInformationSuperStackView.spacing = 50
+        
+        orgInformationStackView1.axis = .horizontal
+        orgInformationStackView1.alignment = .center
+        orgInformationStackView1.distribution = .fillEqually
+        orgInformationStackView1.spacing = 30
+        
+        orgInformationStackView2.axis = .horizontal
+        orgInformationStackView2.alignment = .center
+        orgInformationStackView2.distribution = .fillEqually
+        orgInformationStackView2.spacing = 30
+
+        orgInformationStackView3.axis = .horizontal
+        orgInformationStackView3.alignment = .center
+        orgInformationStackView3.distribution = .fillEqually
+        orgInformationStackView3.spacing = 30
+
+        orgInformationImage1.image = UIImage(.img_reviewfriends_together)
+        orgInformationImage1.contentMode = .scaleAspectFit
```

Comment:
p1;

얘네도 그림자가 필요합니다!
```suggestion
        orgInformationImage1.image = UIImage(.img_reviewfriends_together)
        orgInformationImage1.contentMode = .scaleAspectFit
        orgInformationImage1.layer.applyFigmaShadow(
            color: .black,
            opacity: 0.1,
            xCoord: 0,
            yCoord: 0,
            blur: 5,
            spread: 0
        )
```

Reply:
[1ecc2d9](https://github.com/DeveloperAcademy-POSTECH/MacC-TEAM-ZESTY/pull/70/commits/1ecc2d99c0ef81ebf96c06aa150bcb25a2549d45) 눈이 침침해서 그림자를 깜빡했네요 하하하